examples/opensora_hpcai/README.md

```diff
@@ -464,12 +464,14 @@ video_embed_folder
 
 ## Training
 
-### Open-Sora 1.1 Training
+### Open-Sora 1.1 and above
```

Comment:
Although it may look redundant, it will be more clear to split into Open-Sora 1.2 and Open-Sora 1.1

Reply:
Done